README.md

```diff
@@ -79,14 +78,15 @@ All partitions created by OPL will be 128Mb (it is not recommended to enlarge pa
 
 <details>
   <summary> <b> USB </b> </summary>
-<p>
 
-Game files on USB must be perfectly defragmented either file by file or
-by whole drive, and Dual Layer DVD9 images must be split to avoid the 4GB
-limitations of the FAT32 file system. We do not recommend using any programs.
-The best way for defragmenting - copy all files to pc, format USB, copy all files back.
+
+Game files on USB must be ideally defragmented either file by file or by whole drive,
```

Comment:
Actually it is no more a case, games can be fragmented up to 64 fragments per game. While fragmentation still can affect performance, games will work just fine.

Reply:
> While fragmentation still can affect performance, games will work just fine.

That's why I changed to *ideally*

Reply:
then it cannot be used with _must_  wording :D